src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -14,10 +14,6 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
   display: flex;
   align-items: center;
 
-  &:not(:first-child) {
-    margin-top: spacing(tight);
-  }
-
```

Comment:
Making these consistent is a little tricky because they're different beasts.

There is an 8px distance between text field and its label. I think the single thumb should have the same distance, so I removed the top margin.

![screen shot 2019-02-08 at 6 41 41 pm](https://user-images.githubusercontent.com/6239480/52515520-84f02280-2bd1-11e9-86ce-16f6ec0852de.jpg)
![screen shot 2019-02-08 at 6 41 57 pm](https://user-images.githubusercontent.com/6239480/52515521-84f02280-2bd1-11e9-9f6d-2b24ffc97ad1.jpg)